dtox.sh

```diff
@@ -28,6 +28,7 @@ elif [[ "${BASE_MODE}" == "pull" ]]; then
 fi
 
 USER_INPUT=(
+  "${BASE_INPUT[@]}"
```

Comment:
Using `./dtox.sh inspect` then `python3.12 -V` in the inspect shell was not getting me 3.12.0rc3! Old bug.